README.md

```diff
@@ -0,0 +1,29 @@
+# PatchKit
+
+[![CI Status](https://img.shields.io/travis/omarrasheed/PatchKit.svg?style=flat)](https://travis-ci.org/omarrasheed/PatchKit)
```

Comment:
oh i see did you add it b/c we made it a public cocoapod? if we're only using it internally, we prob don't even have to publish it